utils/zerovec/src/map/vecs.rs

```diff
@@ -96,13 +112,15 @@ pub trait MutableZeroVecLike<'a, T: ?Sized>: ZeroVecLike<'a, T> {
     ///
     /// These are useful to ensure serialization parity between borrowed and owned versions
     fn from_borrowed(b: Self::BorrowedVariant) -> Self;
+
+    /// Convert an owned value to a borrowed T
+    fn owned_as_t(o: &Self::OwnedType) -> &T;
```

Comment:
Thought/Suggestion: Could this be `Self::OwnedType: AsRef<T>` instead?

Reply:
AsRef isn't blanket impls